documentation/design_docs/getting-started/openshift.adoc

```diff
@@ -43,135 +43,6 @@ up EnMasse will take a while, usually depending on how fast it is able
 to download the docker images for the various components. In the
 meantime, you can start to create your address configuration.
 
-[[deploying-enmasse-manually]]
```

Comment:
Personally I think the description of how to install manually is valuable. I agree it may not be relevant to a getting started guide, but it might be worth keeping it for more indepth readers. Though you can always read the script, its nice to see not only the manual steps but also their purpose, as a way of understanding how it all works.

Reply:
@grs I agree, but it's a bit hard to maintain those steps and making them up to date. Btw, you can actually generate the list of manual steps by passing -g to the deployment script. Maybe mentioning that should be sufficient? Another approach would be if we as part of build ran the script with -g, put the output into a manual_steps.adoc, and referred to that.

Wdyt?